packages/block-library/src/buttons/edit.native.js

```diff
@@ -46,8 +70,27 @@ function ButtonsEdit( {
 		}
 	}, [ sizes ] );
 
+	function onAddNextButton( selectedId ) {
```

Comment:
Really minor, but using `const onAddNextButton = ( selectedId ) => {` here instead would be slightly better as `onAddNextButton` would be defined when this line is evaluated. `function` declaration are ok but they're hoisted to the top of this react component so that might lead to inconsistencies.
Same for `onDelete`

Reply:
Personally, I prefer function declarations for non-callback functions, but I noticed that this function was only used as an argument for `debounceAddNextButton`, so I decided to just have an `onAddNextButton` variable set to the result of `debounce`, with the function passed in as a callback.

I decided `onDelete` didn't add much value (and might have actually just been complicating things by making your eyes jump back up the code just for a single-line function), so I just moved that function inline to where it was used.